docs/changelog.md

```diff
@@ -1,3 +1,8 @@
+## v1.x.x [xxxx-xx-xx]
+
+#### Changed
+* Upgraded to Angular 8 (including compatible TypeScript / CLI versions)
+
```

Comment:
also add, 
`Fixed notification service install and enabled check to get latest state`

>under Fixed